src/Components/Components/test/Routing/RouteTableFactoryTests.cs

```diff
@@ -9,13 +9,13 @@
 
 namespace Microsoft.AspNetCore.Components.Test.Routing
 {
-    public class RouteTableTests
+    public class RouteTableFactoryTests
```

Comment:
If there's any way of having a test to show the new behavior that we *don't* route to components in referenced assemblies, that would be ideal. Maybe it's possible for `Components.Test.csproj` to reference some random other components assembly we happen to have in the solution that contains a component with `@page`.